test/unit/model/helpers/installer-test.js

```diff
@@ -206,6 +206,43 @@ describe('Installer', function() {
     });
   });
 
+  describe('copyFile', function() {
+    let source = path.join('.','someTempFolder', 'somefile');
+    let target = path.join('anInstallFolder', 'target');
+
+    it('should call fs#copy with correct arguments', function() {
+      let stub = sandbox.stub(fs, 'copy').yields();
+
+      return installer.copyFile(source, target)
+      .then(function(result) {
+        expect(stub).to.have.been.calledOnce;
+        expect(stub).to.have.been.calledWith(source, target);
+      });
+    });
+
+    it('should resolve as true when no error occurs', function() {
+      let stub = sandbox.stub(fs, 'copy').yields();
+
+      return installer.copyFile(source, target)
+      .then(function(result) {
+        expect(result).to.equal(true);
```

Comment:
FYI, you can also use this syntax `expect(result).to.be.true`, I would leave it as is in this PR